docs/connectors/io-crd-config/sink-crd-config.md

```diff
@@ -247,15 +247,19 @@ spec:
   replicas: 1
   maxReplicas: 5
   liveness:
-    initialDelaySeconds: 10        # --- [1]
-    periodSeconds: 10              # --- [2]
+    failureThreshold:              # --- [1]
```

Comment:
the `liveness` filed should be under `spec.pod` field

Reply:
And we should also remove `grpcurlPersistentVolumeClaim` related docs since it is removed from code